C/c4.def

```diff
@@ -139,6 +139,7 @@ c4docobs_free
 c4repl_new
 c4repl_newWithSocket
 c4repl_free
+c4repl_start
```

Comment:
This file has a special extra requirement by me that I should probably write as a comment on top of the file, but if this is not going to be auto slurped by the bindings script, then please include it below the line `; C4Tests`

Reply:
Hm. It's probably a good idea to create a binding, since the platform code could benefit from using this. It's a very unlikely race condition, but it could conceivably happen in real life.

Reply:
Ok I will leave it there and file a ticket to make use of it then.